extensionsGallery-insider.json

```
@@ -4767,7 +4767,7 @@
 				{
 					"extensionId": "95",
 					"extensionName": "mongodb-ads-extension",
```

Also while we're here, this should be the name to match what's actually in the package.json (and to make it clear that this isn't a "special" version of the extension. It's exactly the VS Code one)
```suggestion
					"extensionName": "mongodb-vscode",
```